api/v1/controllers/JobApplicationController.js

```diff
@@ -0,0 +1,44 @@
+const bodyParser = require('body-parser');
+
+const services = require('../services');
+
+const middleware = require('../middleware');
+const requests = require('../requests');
+const roles = require('../utils/roles');
+
+const router = require('express').Router();
+
+function isRecruiter(req) {
+  return req.user.get('id') == req.params.id && req.user.hasRole(roles.PROFESSIONALS);
+}
+
+function createApplication(req, res, next) {
+  services.JobApplicationService
+    .createApplication(req.user.get('id'), req.body.applicantId, req.body.comments, req.body.favorite)
+    .then((application) => {
+      res.body = application.toJSON();
+      return next();
+    })
+    .catch((error) => next(error));
+}
+
+function getRecruitersApplicants(req, res, next) {
+  services.JobApplicationService
+    .findByRecruiterId(req.params.id)
```

Comment:
The recruiter probably won't have their own recruiter_id, there should be separate routes for those different user flows

Reply:
The recruiter id is the same as their user id. I just called it recruiter id, so it wasn't confusing with the applicants id